docs/composables/useTheme.md

```
@@ -150,6 +151,7 @@ export default {
 
 ## Spec Configuration
 
-| Function        | Description                  | Default Value                                                         | Allowed Values                                                                |
-|-----------------|------------------------------|-----------------------------------------------------------------------|-------------------------------------------------------------------------------|
-| `setSpecConfig` | Sets the spec configuration. | `{ groupByTags: true, collapsePaths: false, showPathsSummary: true, avoidCirculars: false, lazyRendering: false }` | `{ groupByTags: boolean, collapsePaths: boolean, showPathsSummary: boolean, avoidCirculars: boolean, lazyRendering: boolean }` |
+| Function        | Description                  | Default Value                                                                                                                             | Allowed Values                                                                |
+|-----------------|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| `setSpecConfig` | Sets the spec configuration. | `{ groupByTags: true, collapsePaths: false, showPathsSummary: true, avoidCirculars: false, lazyRendering: false, defaultTag: 'Default' }` | `{ groupByTags: boolean, collapsePaths: boolean, showPathsSummary: boolean, avoidCirculars: boolean, lazyRendering: boolean, defaultTag: string }` |
+```
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)